specification/dartLangSpec.tex

```diff
@@ -13966,8 +15437,10 @@ \subsection{Lexical Lookup}
   (\ref{classes}).
 \item
   When $\ell$ has access to \THIS{}
```

Comment:
Where was `\ell` specified? I haven't expanded *everything*, but I can't find a reference close above (like, in the first 4k lines)

Also what does "has access to \THIS{}" mean? That is accesses `this` or that it *could* access `this` if it wanted to. From the context, it sounds like the former. Consider rewriting to make it clear.